text/0000-long-error-codes-explanation-normalization.md

```diff
@@ -0,0 +1,143 @@
+
+Start Date: 2016-01-04
+
+RFC PR:
+
+Rust Issue: N/A
+
+# Summary
+
+Long error codes explanations haven't been normalized yet. For example, actually, not all error codes explanation provide an erroneous code example or a way to fix the error. This RFC intends to clarify this by providing a template to follow.
```

Comment:
"For example, actually" can become "For example".
"not all error codes explanation" -> "not all error code's explanations"
"This RFC intends to clarify this by providing a template to follow." -> "This RFC intends to clarify long error codes by providing a template to follow".